src/components/loading-indicator/LoadingIndicatorWrapper.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
```

Comment:
I am guessing the diff/rename was too big so git added this as a new file. Compare to older .js version